front/pages/api/v1/w/[wId]/data_sources/[name]/documents/[documentId]/index.ts

```diff
@@ -279,6 +289,33 @@ async function handler(
         });
       }
 
+      const whitelistedWorkspaceIds =
```

Comment:
Bearish on having "experimental product" code in the API. How about we make it another upsertHook, one to update trackers (triggered only if the magic word is in the document), and one to actually try to compute the tracked documents potential updates?

Reply:
as per IRL, we're moving everything to `front` and adding temporal there